docs/api/v1/openapi.yml

```
@@ -945,6 +945,31 @@ paths:
           $ref: '#/components/responses/unauthorized'
         '404':
           $ref: '#/components/responses/notFound'
+    put:
+      x-eov-operation-id: updateVariable
+      x-eov-operation-handler: v1/handlers/variables/variables.handler
+      tags:
+        - Variables
+      summary: Update a variable
+      description: Update a variable from your instance.
```

```suggestion
      description: Update a variable from your instance.
      parameters:
        - $ref: '#/components/parameters/variableId'
```

Just guessing on this one.

---

So Sorry 😞 Totally overlooked that too! Thanks Justin! 🙏